lms/static/sass/course/_info.scss

```diff
@@ -56,6 +61,85 @@ div.info-wrapper {
       font-style: normal;
     }
 
+    .upgrade-banner {
+      // This banner uses the Pattern Library's defined variables
+      border-style: solid;
+      border-color: $uxpl-gray-background $uxpl-gray-background $uxpl-gray-background $uxpl-green-base;
+      border-width: 1px 1px 1px 10px;
+      max-height: 100px;
```

Comment:
This is something I am planning to test again.  This was per the comments from @lizcohen, but I think I might have taken them to literally.  I am working on completing a task that was just unblocked and then I will return to look at this.